### PR TITLE
Adds TravisCI and tests current Ruby versions

### PR DESCRIPTION
**Pull request by danielpowell4**
### Contributor Self-Check:

* [x] My commits and messages [are solid](https://chris.beams.io/posts/git-commit/)
* [x] I have included tests that check success and failure
* [x] I have updated the README as appropriate

### Which GitHub Issues does this PR address?

#11 

### What does this PR do?

The had-been Codeship config had permissions tied to jzornow.

This commit adds Travis. The main benefit here is that we can easily test several versions of ruby with a super simple config. 

### How do I manually test this?

Check that the Travis build passes and has a decent looking config

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.2.6
+2.3.6
```
Comment (toddsiegel) [inline .ruby-version]: i'd say remove this altogether.

File: intacct_ruby.gemspec [hunk 2]
```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |spec|
   spec.version               = IntacctRuby::VERSION
   spec.authors               = ['Jeremy Zornow']
   spec.email                 = ['jeremy@zornow.com']
-  spec.required_ruby_version = '>= 1.9.3'
+  spec.required_ruby_version = '>= 2.2.6'
```
Comment (toddsiegel) [inline intacct_ruby.gemspec]: Any reason though not to support the whole 2.2.0 line? More generally, what should be our general support policy? I'd say any currently supported Rubies.

Comment (danielpowell4) [reply intacct_ruby.gemspec]: I don't think there is a reason to not do the whole 2.2 line.

I picked 2.2.6 as that was the last ruby_version that jz had in here. I bumped from 1.9.3 only so `bundle` would run in CI (pry-byebug wanted 2+)

I agree with "any currently supported Rubies" which would mean we'd be at `2.2.9` as the lowest current. Right?

File: .ruby-version (hunk 1 above)
Comment (danielpowell4) [reply .ruby-version]: I am fine with that. Seems more of a pain than anything.

File: intacct_ruby.gemspec (hunk 2 above)
Comment (toddsiegel) [reply intacct_ruby.gemspec]: Yeah, for another month and half, or as long as we still have an app using it. :wink:

**Pull request merged**